integration-tests/utils/integration-helpers.js

```diff
@@ -169,7 +169,7 @@ const registerUser = withSignedInPage(async ({ page, token }) => {
   await page.evaluate(async () => {
     await window.catchErrorResponse(async () => {
       await window.Ajax().User.profile.set({ firstName: 'Integration', lastName: 'Test', contactEmail: 'me@example.com' })
-      await window.Ajax().User.acceptTos()
+      await window.Ajax().User.acceptSamTos()
```

Comment:
ToS lives in Sam now, so call that endpoint instead. The other endpoint is still around due to backwards compatibility that is necessary for the complex rollout of these changes. Eventually the old ToS code will be removed.

Reply:
How do we remember to remove the old ToS code when the time comes? Does it make sense to add a comment referencing the future removal to where `Ajax().User.acceptTos()` is defined?

Reply:
Workspaces team has a ticket in their backlog to clean this up once the old way of handling ToS is fully phased out. See [WOR-153](https://broadworkbench.atlassian.net/browse/WOR-153)